### PR TITLE
Add styling for active sort column

### PR DESCRIPTION
**Pull request by payten**

### CODE REVIEW

**Comment by payten** [comment]
Also patched a few issues in the grade item headers.  Namely the title was showing 'undefined' and the sort link had the text 'Assignment title' for all assignments.

File: tool/src/java/org/sakaiproject/gradebookng/tool/panels/AssignmentColumnHeaderPanel.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	
 	<div class="gb-title">
 		<span wicket:id="externalAppFlag"></span>
-		<a wicket:id="title">Assignment title</a>
+		<a wicket:id="title"><wicket:container wicket:id="label" /></a>
```
Comment (steveswinsburg) [inline tool/src/java/org/sakaiproject/gradebookng/tool/panels/AssignmentColumnHeaderPanel.html:9]: I should use this container tag more, so used to just sticking it on span's. I accidentally broke this with the addition of the link, and staring at the UI for so long the 'Assignment title' looked just fine! Good one.

File: tool/src/webapp/scripts/gradebook-grades.js [hunk 2]
```diff
@@ -1458,7 +1458,11 @@ GradebookHeaderCell.prototype.getCategorizedOrder = function() {
 
 GradebookHeaderCell.prototype.setupTooltip = function() {
   if (this.$cell.hasClass("gb-grade-item-column-cell")) {
-    var $title = this.$cell.find(".gb-title > span");
+    var $title = this.$cell.find(".gb-title > a");
+    if ($title.length == 0) {
+        $title = this.$cell.find(".gb-title > span");
```
Comment (steveswinsburg) [inline tool/src/webapp/scripts/gradebook-grades.js]: With the container, is there still a span?

Comment (payten) [inline tool/src/webapp/scripts/gradebook-grades.js]: Yeah, good catch! It should always be `> a`.

**Comment by payten** [comment]
Fixed up.  Thanks!

**Pull request merged**